content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -19,7 +19,7 @@ To run node conformance test, a node must satisfy the same prerequisites as a
 standard Kubernetes node. At a minimum, the node should have the following
 daemons installed:
 
-* Container Runtime (Docker)
+* Container Runtime (containerd, crio etc.)
```

Comment:
```suggestion
* Container Runtime (containerd, crio )
```
Take a look at [this](https://github.com/kubernetes/website/issues/38898#issuecomment-1380250451) as well